### DISCUSSION TITLE
Interesting A/B Flags

### DISCUSSION THREAD

**Discussion by arichornlover** [PoomSmart/YTABConfig#31] [General]
This flag came with YouTube v17.43.1 which is called `videoZoomFreeZoomEnabled`

**this is what videoZoomFreeZoomEnabled does:**
This flag lets you zoom into YouTube videos 8x more on the iPhones & iPads and it does replace the old zoom feature kind of.

more interesting flags can be suggested below!

**Comment by PoomSmart** [comment]
Diff between the latest version and the previous version doesn't include that flag and I remember that flag exists for previous versions too.

**Comment by arichornlover** [comment]
This flag came with YouTube v17.xx.x which is called
`iosColorizedProgressBar`
A very unique A/B Feature Flag that changes the color of the video player bar! It will be White instead of Red - it does turn red time to time but dynamically changes to white when not interacting with the bar

**Reply by arichornlover** [reply]
If you’re using YouTube 18.xx.x or later then there may be a chance that this flag won’t work. Because for me I had no luck to getting it to work now once enabled in A/B settings.

**Comment by arichornlover** [comment]
I found another interesting flag that also came with YouTube v17.xx.x which is called
`channelsClientConfigIosChannelNavRestructuring`
Turning on this flag gives you the Redesigned Channel Menu on the YouTube app.
Also YouTube did enable this flag to some users by default so if you turn off this flag you will get the old Channel Menu.

**Comment by arichornlover** [comment]
Alright here’s some more interesting flags I found that may help you out on the YouTube App. Also must be using YouTube v17.49.6 or v18.06.6 to have some access to some of these flags!!

`iosFixHeaderviewStatusbarHeight` - This flag fixes the issue on the YouTube app if you watch the app in portrait/landscape mode. With this `Enabled` then the video player will be fixed and positioned correctly. ✅

`iosEnableStaticShortsSearchButton` - the search button that’s shown in the Shorts Tab. I cannot confirm that this really disabled it. It may require another flag to have the search button in shorts fully `Disabled`. 😕

**This Flag below requires v18.04.3-newer, sorry!**
`iosDisableSearchPredictiveText` -  this flag disables the Predictive text in the app. Having this `Enabled` will disable it hopefully.

**Comment by arichornlover** [comment]
This Flag is available for any YouTube Version I think? but may be very important flag through out the app.
`ShouldBlockUpgradeDialog`should be `enabled` since it will prevent YouTube from prompting you with the Update Dialog even if they try to enforce it on you. If this doesn’t block the Dialog then disable the flags shown below
`shouldForceUpgrade` = `disabled`
`shouldShowUpgrade` = `disabled`
`shouldShowUpgradeDialog` = `disabled`

**Reply by Landxhent** [reply]
Doesn't work for me. Have it right, but still have the dialog. 16.45.4
